app/src/main/res/values/dimens.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <!--  My profile  -->
     <dimen name="margin_header_content">16dp</dimen>
```

Comment:
very very very many values
you need to combine similar ones

Reply:
Some dimensions were removed

Reply:
https://github.com/greezun/task3/blob/master/app/src/main/res/values/dimens.xml

this is not standard, but example
resources must be universal

Reply:
Shorted dimens.xml